### PR TITLE
Update readme & user guide

### PR DESCRIPTION
**Pull request by jkottiku**
Added description for GPU specific configuration files and its usage.

### CODE REVIEW

**Review by manoj-freyr — APPROVED** [review]
looks good

**Review by lpaoletti — APPROVED** [review]

File: docs/ug1main.md [hunk 1]
```diff
@@ -51,20 +52,25 @@ RVS package components are installed in `/opt/rocm`. Package contains:
     Command examples
     ./rvs --help ; Lists all options to run RVS test suite
     ./rvs -g ; Lists supported GPUs available in the machine
-    ./rvs -d 3 ; Run set of RVS sanity tests (in rvs.conf) with verbose level 3
-    ./rvs -c conf/gst_single.conf ; Run GST module tests
+    ./rvs -d 3 ; Run set of RVS default sanity tests (in rvs.conf) with verbose level 3
+    ./rvs -c conf/gst_single.conf ; Run GST module default test configuration
 
-#### Run version pre-complied and packaged with ROCm release
+### Run version pre-complied and packaged with ROCm release
```
Comment (peterjunpark) [inline docs/ug1main.md:58]:
```suggestion
### Run version pre-compiled and packaged with ROCm release
```

**Comment by peterjunpark** [comment]
Hi @jkottiku ,

What is the difference between `ug1main.md` and this file? https://github.com/ROCm/ROCmValidationSuite/blob/master/docs/install/installation.rst?plain=1

`installation.rst` is what's published on the docs site. I think your updates should go there instead

**Comment by peterjunpark** [comment]
Also, I'm not sure what this is about: `installarchive.rst`.
https://github.com/ROCm/ROCmValidationSuite/blob/master/docs/install/installarchive.rst
I'm assuming this is deprecated documentation? Is there a reason to keep this around -- not sure if this is confusing to users

**Comment by jkottiku** [comment]
> Hi @jkottiku ,
> 
> What is the difference between `ug1main.md` and this file? https://github.com/ROCm/ROCmValidationSuite/blob/master/docs/install/installation.rst?plain=1
> 
> `installation.rst` is what's published on the [docs site](https://rocm.docs.amd.com/projects/ROCmValidationSuite/en/latest/install/installation.html). I think your updates should go there instead

ug1main.md - User guide in github (https://github.com/ROCm/ROCmValidationSuite/blob/master/docs/ug1main.md) 
docs site - Was done recently by Roopa. We are unaware how it works exactly. 

Updated installation.rst as well.

**Pull request merged**